ledger/src/blockstore_processor.rs

```diff
@@ -949,6 +971,14 @@ pub fn confirm_slot(
 
     timing.execute_timings.accumulate(&execute_timings);
 
+    if !check_result.finish_verify() {
+        warn!("Ledger proof of history failed at slot: {}", slot);
+        return Err(BlockError::InvalidEntryHash.into());
+    }
+
+    // todo: is this correct?
+    let transaction_duration_us = check_result.verify_duration_us;
```

Comment:
this looks correct, what was your concern?

Reply:
Yup, just an old TODO that a forgot to remove.